src/components/selector-dropdown/SelectorDropdown.js

```diff
@@ -23,6 +26,8 @@ type Props = {
     dividerIndex?: number,
     /** Options to keep the results always open */
     isAlwaysOpen?: boolean,
+    /** Option to disable dynamic positioning with popper */
+    isPopperDisabled?: boolean,
```

Comment:
should we call this `hasDynamicPosition`?

Reply:
I renamed and updated the corresponding code.